configs/common.conf

```diff
@@ -47,6 +47,15 @@ send-keys 'ros2 security generate_artifacts -k keystore -p policies/tb3_gazebo_p
 split-window -v
 send-keys 'env | grep ROS' Enter
 
+new-window -n reconnaissance
+send-keys 'aztarna -t ros2'
+
+split-window -h
+send-keys 'tshark -i eth0 -f "udp" -Y "rtps"' Enter
```

Comment:
I don't think we want to run the command by default for every demo. Perhaps remove the Enter, and remind the reader to press enter in the command window when starting the tutorial. Or add this window using a separate root level config, like we have for secure and unsecure.

Reply:
That's fair. Let me remove it then.

Reply:
9b76724e1586e2a0fc28d938bb7da7fbb6dcb8f0